.github/workflows/bsim-tests.yaml

```
@@ -163,8 +163,12 @@ jobs:
       - name: Merge Test Results
         run: |
           pip3 install junitparser junit2html
-          junitparser merge ./bsim_*/*bsim_results.*.xml ./twister-out/twister.xml junit.xml
-          junit2html junit.xml junit.html
+          xml_files=`find  twister-out bsim_out -name "twister.xml" -or -name "*bsim_results*xml" 2> /dev/null`
```

It's very unlikely to create an error but it's maybe safer to add `.xml` and not just `xml` for the bsim results.

```suggestion
          xml_files=`find  twister-out bsim_out -name "twister.xml" -or -name "*bsim_results*.xml" 2> /dev/null`
```

---

Thanks changed